DTS-STN/secure-client-hub
# Update Main from dev November 2023

### Description of proposed changes:
This PR serves to update our Main branch to be in line with dev as of November 14, 2023. Necessary to ensure this properly builds on DSHP.

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Ensure everything works as expected
